code/modules/reagents/reagent_containers/food/nutrient_blocks.dm

```diff
@@ -84,5 +84,5 @@
 /obj/item/weapon/storage/fancy/nutricube_tray/filled
 	name = "Nutricube Storage Tray"
 	desc = "A tray full of nutrient cubes, this is much heavier than it looks."
-	icon_state = "tray"
+	icon_state = "tray8"
```

Comment:
I tried fixing this before. It fixes it on the map, breaks it in the game. Could you revert this?

Reply:
What happened? I didn't notice a problem during testing.

Reply:
It was only for a round.

The nutristorage sprite is dynamically called on item removal or addition. So changing it to a hardcoded value like 8, will turn it into tray88 in game. Or tray87. Obviously those don't exist.

I have a different fix in mind, though. One that shouldn't fuck with it.

Reply:
Instead of changing it in code, just copy over a desired sprite, and call it tray in that file. Check it out in-game. Should work.